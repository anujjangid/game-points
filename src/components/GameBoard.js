import React from 'react';
import Players from './Players';
import Score from './Score';

const GameBoard = () => {
  return (
    <React.Fragment>
      {/* player block section  */}
      <Players />
      {/* score section */}
      <Score />
    </React.Fragment>
  );
};

export default GameBoard;
