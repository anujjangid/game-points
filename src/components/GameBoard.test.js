import React from 'react';
import { shallow } from 'enzyme';

import GameBoard from './GameBoard';

describe('Test suite for GameBoard Component', () => {
  // create a UI snapshot to indentify UI changes or design regressions
  const component = shallow(<GameBoard />);
  it('render correctly text component', () => {
    expect(component).toMatchSnapshot();
  });
  it('should render one div', () => {
    expect(component.find('div')).toHaveLength(0);
  });
});
