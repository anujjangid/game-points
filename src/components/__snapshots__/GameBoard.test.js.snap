// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test suite for GameBoard Component render correctly text component 1`] = `ShallowWrapper {}`;
